utils/validationSchemas.ts

```diff
@@ -47,3 +47,12 @@ export const registerValidationSchemaThirdForm = object({
     "Entrez le même mot de passe que le premier champ",
   ),
 });
+
```

Comment:
is these lines related to the donation form ?

Reply:
No...
I need a new validation schema for each categorie, right ?

Reply:
Yes I think